app/admin/models/Mealtimes_model.php

```diff
@@ -50,13 +50,13 @@ public function isAvailable($datetime = null)
         }
 
         return $datetime->between(
-            Carbon::createFromTimeString($this->start_time),
-            Carbon::createFromTimeString($this->end_time)
+            Carbon::createFromFormat('Y-m-d H:i:s', $datetime->format('Y-m-d ').$this->start_time),
+            Carbon::createFromFormat('Y-m-d H:i:s', $datetime->format('Y-m-d ').$this->end_time)
         );
     }
 
     public function isAvailableNow()
     {
         $this->isAvailable();
```

Comment:
Just saw this lol, shouldn't it return the value from isAvailable() method.

Reply:
Oops.